### PR TITLE
Add border between book title and book info (Book details part 3)

### PR DESCRIPTION
**Pull request by JuanGutierrez43**
## Summary 

This will add the border between the title and the book info to be the presentation more fresh and more structured. For this I used the rectangle image from the invision, and flex to set it on the page.

**New features**

- Add image rectangle.png
- Set the image to the book portrait using flex

## Screenshots 
Screenshot from 2020-03-05 11-13-54

## Cards

https://trello.com/c/kQ206RUk/28-detalle-de-libro-iii-estructura

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
         <h1 class="book-title">Título del libro</h1>
         <p class="book-genre">(género)</p>
       </header>
+      <img class="rectangle" src="images/rectangle.png" alt="Title Border">
```
Comment (eledwinn) [inline index.html]: You can to use border instead image, avoid to use html elements as decorative details as borders, shadows, etc.

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 2]
```diff
@@ -53,3 +57,8 @@
   font-weight: 700;
   margin-right: 5px;
 }
+
+.rectangle {
```
Comment (CrisVega08) [inline scss/detail.scss]: I am agree with edwin comment so change it and delete this style.

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 3]
```diff
@@ -20,6 +20,12 @@
   line-height: 30px;
 }
 
+.book-header {
+  margin: 10px 0 15px 15px;
+  border-bottom: 5px solid $earls-green;
+  width: 550px;
```
Comment (eledwinn) [inline scss/detail.scss]: Is this width necessary?

**Review by eledwinn — CHANGES_REQUESTED** [review]
Please add shadow in the card.

File: scss/detail.scss [hunk 4]
```diff
@@ -10,36 +10,46 @@
 
 .book-cover-page {
   display: flex;
-  margin-top: 10px;
+  margin: 50px 0 0 50px;
+  height: 416px;
+  width: 900px;
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
```
Comment (CrisVega08) [inline scss/detail.scss]: create a variable on colors for this.

File: scss/detail.scss [hunk 5]
```diff
@@ -52,4 +62,4 @@
   color: $black;
   font-weight: 700;
   margin-right: 5px;
-}
+}
```
Comment (CrisVega08) [inline scss/detail.scss]: Add empty line

File: scss/detail.scss [hunk 6]
```diff
@@ -10,36 +10,46 @@
...
   display: flex;
-  margin-top: 10px;
+  margin: 50px 0 0 50px;
+  height: 416px;
+  width: 900px;
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
 }
 
+//div super gigante que englobe al cover-page
```
Comment (CrisVega08) [inline scss/detail.scss]: English please

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

**Review by eledwinn — APPROVED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**